lua/terrortown/lang/zh_hans.lua

```diff
@@ -1868,114 +1868,114 @@ L.search_dmg_teleport = "看起来他的DNA以超光速粒子之形式散乱在
---L.search_floor_eggshell = "Disgusting looking white specks cover the body of the victim. It looks like egg shells."
---L.search_floor_flesh = "The victim's clothing feels kinda moist. As if they fell onto a wet surface. Like a fleshy surface, or the sandy ground of a water body."
---L.search_floor_grate = "The skin of the victim looks like a steak. Thick lines arranged in a grid are visible all over them. Did they rest on a grate?"
---L.search_floor_alienflesh = "Alien flesh, you think? Sounds kinda outlandish. But your detective helper book lists it as a possible floor surface."
---L.search_floor_snow = "On first glance their clothing only feels wet and ice-cold. But once you see the white foam on the rims you understand. It's snow!"
---L.search_floor_plastic = "'Ouch, that has to hurt.' Their body is covered in burns. They look like those you get when sliding over a plastic surface."
---L.search_floor_metal = "At least they can't get tetanus now that they are dead. Rust covers their wounds. They probably died on a metal surface."
---L.search_floor_sand = "Small little rough rocks are stuck to their cold body. Like coarse sand from a beach. Argh, it gets everywhere!"
---L.search_floor_foliage = "Nature is wonderful. The victim's bloody wounds are covered with enough foliage that they are almost hidden."
---L.search_floor_computer = "Beep-boop. Their body is covered in computer surface! How does this look, you might ask? Well, duh!"
---L.search_floor_slosh = "Wet and maybe even a bit slimy. Their whole body is covered with it and their clothes are soaked. It stinks!"
---L.search_floor_tile = "Small shards are stuck to their skin. Like shards from floor tiles that shattered on inpact."
---L.search_floor_grass = "It smells like fresh cut grass. The smell almost overpowers the smell of blood and death."
---L.search_floor_vent = "You feel a fresh gust of air when feeling their body. Did they die in a vent and take the air with them?"
---L.search_floor_wood = "What's nicer than sitting on a hardwood floor and dwelling in thoughts? At least lot lying dead on a wooden floor!"
---L.search_floor_default = "That seems so basic, so normal. Almost default. You can't tell anything about the kind of surface."
---L.search_floor_glass = "Their body is covered with many bloody cuts. In some of them glass shards are stuck and look rather threatening to you."
---L.search_floor_warpshield = "A floor made out of warpshield? Yep, we are as confused as you were. But our notes clearly state it. Warpshield."
-
---L.search_water_1 = "The victim's shoes are wet, but the rest seems dry. They were probably killed with their feet in water."
---L.search_water_2 = "The victim's shoes are trousers are soaked through. Did they wander through water before they were killed?"
---L.search_water_3 = "The whole body is wet and swollen. They probably died while they were completely submerged."
+L.search_floor_antlions = "尸体上仍然满是蚁狮。地板上一定覆盖着它们。"
```

Comment:
`search_floor` phrases have an issue with pronouns - see review description.

Reply:
So does this need to be modified :)

Reply:
Yes but I'd like to hear feedback from some other devs on the matter before I write it.

Reply:
Okay

Reply:
@TimGoll :)